examples/emr/v1beta1/instancefleet.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: emr.aws.upbound.io/v1beta1
+kind: InstanceFleet
+metadata:
+  annotations:
+    meta.upbound.io/example-id: emr/v1beta1/instancefleet
+  labels:
+    testing.upbound.io/example-name: task
+  name: task
+spec:
+  forProvider:
+    clusterIdRef: example-cluster
```

Is this `clusterId: example-cluster` or below?

```suggestion
    clusterIdRef: 
      name: example-cluster
```

---

Mistake